packages/mdc-typography/_mixins.scss

```diff
@@ -23,11 +23,12 @@
 @use "sass:list";
 @use "sass:map";
 @use "sass:string";
-@use "@material/feature-targeting" as feature-targeting;
```

Comment:
Have we tried dropping the "as feature-targeting"? I don't think it's necessary.

Reply:
I didn't I was following the way we currently have it implemented in the rest of the mixin files.

Reply:
It was because copybara changes it to "featuretargeting" internally, which breaks the usage of it in the file without an explicit `as`